doc/sphinx/changes.rst

```diff
@@ -8,6 +8,515 @@ Recent changes
+  (`#15650 <https://github.com/coq/coq/pull/15650>`_,
+  fixes `#15600 <https://github.com/coq/coq/issues/15600>`_,
+  by Enrico Tassi).
+- **Added:**
+  :cmd:`Print Notation` command that prints the level and
+  associativity of a given notation definition string
+  (`#15683 <https://github.com/coq/coq/pull/15683>`_,
+  fixes `#14907 <https://github.com/coq/coq/issues/14907>`_
+  and `#4436 <https://github.com/coq/coq/issues/4436>`_
+  and `#7730 <https://github.com/coq/coq/issues/7730>`_,
+  by Ali Caglayan and Ana Borges, with help from Emilio Jesus Gallego Arias).
+- **Added:**
+  a warning when trying to deprecate a definition
+  (`#15760 <https://github.com/coq/coq/pull/15760>`_,
+  by Pierre Roux).
+- **Added:**
+  A deprecation warning that :n:`>` in a field of a :cmd:`Class`
+  and the :g:`Class >` syntax will in the future also
+  declare :ref:`coercions <coercions>` as they do when used in
+  :cmd:`Record` commands
+  (`#15802 <https://github.com/coq/coq/pull/15802>`_,
+  addresses `#2643 <https://github.com/coq/coq/issues/2643>`_
+  and `#9014 <https://github.com/coq/coq/issues/9014>`_,
+  by Pierre Roux, reviewed by Gaëtan Gilbert, Ali Caglayan,
+  Jason Gross, Jim Fehrle and Théo Zimmermann).
+- **Added:**
+  the :attr:`nonuniform` boolean attribute that silences the
+  non-uniform-inheritance warning when user needs to declare such a
+  coercion on purpose
+  (`#15853 <https://github.com/coq/coq/pull/15853>`_,
+  by Pierre Roux, reviewed by Gaëtan Gilbert and Jim Fehrle).
+- **Added:** All commands which can import modules (e.g. ``Module
+  Import M.``, ``Module F (Import X : T).``, ``Require Import M.``,
+  etc) now support :token:`import_categories`. :cmd:`Require Import`
+  and :cmd:`Require Export` also support :token:`filtered_import`.
+  (`#15945 <https://github.com/coq/coq/pull/15945>`_, fixes `#14872
+  <https://github.com/coq/coq/issues/14872>`_, by Gaëtan Gilbert).
+- **Fixed:**
+  Make `Require Import M.` equivalent to `Require M. Import M.`
+  (`#15347 <https://github.com/coq/coq/pull/15347>`_,
+  fixes `#3556 <https://github.com/coq/coq/issues/3556>`_,
+  by Maxime Dénès).
+
+Command-line tools
+^^^^^^^^^^^^^^^^^^
+
+- **Added:**
+  coq_makefile variable `COQPLUGININSTALL` to configure the installation of ML plugins
+  (`#15788 <https://github.com/coq/coq/pull/15788>`_,
+  by Cyril Cohen and Enrico Tassi).
+- **Added:**
+  Added -bytecode-compiler (yes|no) flag for ``coqchk`` enabling
+  :tacn:`vm_compute` during checks, which is off by default.
+  (`#15886 <https://github.com/coq/coq/pull/15886>`_,
+  by Ali Caglayan).
+- **Fixed:**
+  :n:`coqdoc` confused by the presence of command :n:`Load` in a file
+  (`#15511 <https://github.com/coq/coq/pull/15511>`_,
+  fixes `#15497 <https://github.com/coq/coq/issues/15497>`_,
+  by Hugo Herbelin).
+
+CoqIDE
+^^^^^^
+
+- **Changed:**
+  Bumped lablgtk3 lower bound to 3.1.2
+  (`#15947 <https://github.com/coq/coq/pull/15947>`_,
+  by Pierre-Marie Pédrot).
```

Comment:
Since this is the only item in CoqIDE, we could also move this to the "Infrastructure and dependencies" section.